packages/webpack/src/utils/with-nx.ts

```diff
@@ -164,7 +164,7 @@ export function withNx(opts?: { skipTypeChecking?: boolean }) {
       profile: options.statsJson,
       resolve: {
         ...config.resolve,
-        extensions,
+        extensions: [...extensions, ...config.resolve.extensions],
```

Comment:
Because of fun JS reasons, this will error out if `config.resolve.extensions` is undefined.

needs to be `[...extensions, ...(config.resolve.extensions || [])]`